src/components/button/button.test.ts

```diff
@@ -45,7 +45,7 @@ describe('<sl-button>', () => {
 
     it('passes accessibility test', async () => {
       const el = await fixture<SlButton>(html` <sl-button disabled>Button Label</sl-button> `);
-      await expect(el).to.be.accessible();
```

Comment:
It's not obvious, but `await` is required here, otherwise you'll get an error like this:

```
An error was thrown in a Promise outside a test. Did you forget to await a function or assertion?
      Error: Error: No elements found for include in page Context
        at node_modules/chai-a11y-axe/src/accessible.js:36:16
```

Not sure if it's been removed from other locations.